test/distributed/fsdp/test_fsdp_misc.py

```diff
@@ -606,6 +606,7 @@ def test_homogeneous_attributes(self):
         all_attr_name_and_values = [
             ("_use_orig_params", False, True),
             ("limit_all_gathers", False, True),
+            ("_use_full_prec_in_eval", False, True),
```

Comment:
@awgu this is an attr that should be set uniformly for all handles, but isn't a direct ctor input. So I added the below logic to test, LMK if shoudl do something different.